packages/discord.js/src/structures/CommandInteraction.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const { Collection } = require('@discordjs/collection');
+const { ApplicationCommandOptionType } = require('discord-api-types/v9');
```

Comment:
why?

Reply:
I thought this was necessary to use these `@link`s
![image](https://user-images.githubusercontent.com/79790819/151720585-3c234d20-0773-44d6-8d25-56736377a2c3.png)

Reply:
it isn't

Reply:
Ok, thanks for let me know.
I will fix it as soon as I can.